PhysicsTools/NanoAOD/python/jets_cff.py

```diff
@@ -656,12 +664,15 @@
 from RecoJets.JetProducers.QGTagger_cfi import  QGTagger
 qgtagger=QGTagger.clone(srcJets="updatedJets",srcVertexCollection="offlineSlimmedPrimaryVertices")
 
+from RecoJets.JetProducers.hfJetShowerShape_cfi import  hfJetShowerShape
+hfjetshowershape=hfJetShowerShape.clone(theJets="updatedJets",theVertices="offlineSlimmedPrimaryVertices")
```

Comment:
isn't this supposed to be added only with an era, to cover just the cases where the `hfJetShowerShape` is not available?
Or is there a need to always rerun this even if the nanoAOD is made from scratch (from steps starting with SIM or RAW)?